src/app/app-interactive-map/store/interactive-map.effects.ts

```diff
@@ -36,8 +37,29 @@ const preferredMaps = [
 const preferredMap = (mapItems: MapItem[]): MapItem =>
   mapItems.find((mapItem) => preferredMaps.includes(mapItem.name)) || mapItems[0];
 
+const preferredSpecies = [
+  'Escherichia coli',
+];
+
+const preferredSpecie = (species: Specie[]): Specie =>
```

Comment:
Please create a utility function to avoid duplication.

const preferredItem = <T>(preferredItems: T, key: keyof T) => (items: T[]): T => items.find((item) => preferredItems.includes(item[key])) || items[0];

Or something like this (I think typing is not perfect here)